e2e-tests/cypress/support/appeal-header-footer/confirmTermsAndConditionsLinkDisplayed.js

```diff
@@ -1,7 +1,7 @@
 module.exports = () => {
   cy.get('[data-cy="Terms and conditions"]')
     .should('have.attr', 'href')
-    .and('include', 'https://www.gov.uk/government/publications/appeals-portal-documents');
+    .and('include', 'https://www.gov.uk/guidance/appeals-casework-portal-terms-and-conditions');
```

Comment:
is that URL definitely correct - it's 404 for me

Reply:
Definitely correct and not 404. Not sure why you are seeing that? The orignal url is 404.

Reply:
no, wait, I must have messed up - it's good

Reply:
Fairly sure it's because I copied in the closing quote like an absolute chimp :/